### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by estebanmual**
In this Milestone I: 
  - Initialised React app.
  - Created a directory for the components.
  - Added React Router and set two Routes and Links for the app's navigation
  - Created a component called Book for displaying a single book
  - Created a form for adding a book.
  - Display a button "Check status" only on the categories page

### CODE REVIEW

File: src/components/AddBook.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <div>
+      <h2>ADD NEW BOOK</h2>
+      <div>
+        <input type="text" placeholder="Book title" />
+        <input type="text" multiple list="drawCategories" placeholder="Category" />
+        <datalist id="drawCategories">
+          <option>Action</option>
+        </datalist>
+        <button type="button">ADD BOOK</button>
+      </div>
```
Comment (FrederickMih) [inline src/components/AddBook.jsx]: - [x] Please let's stick to the requirements of this project which says this should be a `form` (not a `div` as in your case) with inputs(You have thems already) for the title :+1: and author(missing) of the book. Let's also use `submit` as the value of the button type ie `<button type = "submit".....

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @estebanmual, 

You have done a fantastic job so far!
However, they are some issues that you still need to work on to go to the next project but you are almost there!


# To Highlights :rocket: 

- GitHub/Git workflow has been followed :100: 
- Linters are running and all checks passed :heavy_check_mark: 
- Pull Request has a descriptive title and summary :+1: 

## Required Changes ♻️

- [x] Kindly update your README file to match your project. Let's make it descriptive. All the main headers from [this template](https://github.com/microverseinc/readme-template) should be present in your repo.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.jsx (hunk 1 above)
Comment (estebanmual) [reply src/components/AddBook.jsx]: Thank you so much for your feedback @FrederickMih! I really appreciate your comments looking to improve my project. 😄

File: README.md [hunk 2]
```diff
@@ -1,70 +1,49 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (Meltrust) [inline README.md]: According to the Microverse Readme template, your repo readme is lacking this section:

image

Therefore, as requested by the previous reviewer:

- [x] Kindly provide the "Getting started" section on your readme with instructions about how to clone, install and run your project locally.

## Explanation

On the Microverse Readme Template, nothing is optional except for the "Tests" and "Live Demo" sections if not required from the project.

## Hints and further advice

You can create a great readme that meets all requirements once, and then use it for all your projects, just customizing it each time!

That way, we will meet this requirement and will be ready for approval 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @estebanmual,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

You can do it

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (estebanmual) [reply README.md]: Sorry about that, I totally forgot that section. Thank you so much for your review @Meltrust

**Review by Meltrust — APPROVED** [review]
Hi @estebanmual,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**